File: src/TCC.Contabilidade.Domain/Entities/Empresa.cs
namespace TCC.Contabilidade.Domain.Entities;

public class Empresa
{
    public Guid Id { get; set; }

    public string Nome { get; set; } = string.Empty;

    public string CNPJ { get; set; } = string.Empty;

    public CompanyConfig? Config { get; set; }

    public ICollection<UsuarioEmpresa> UsuariosEmpresas { get; set; } = new List<UsuarioEmpresa>();
}

File: src/TCC.Contabilidade.Web/src/interfaces/empresa.ts
export interface Empresa {
  id: string;
  nomeFantasia: string;
  cnpj: string;
}

export interface CreateEmpresaRequest {
  nome: string;
  cnpj: string;
}

export interface UpdateEmpresaRequest {
  nome: string;
  cnpj: string;
}
